app/views/dashboard/ng-partials/_devices.html.haml

```diff
@@ -14,29 +14,31 @@
               %input{placeholder: "Brocolli Overlord", type: "text", ng_model: 'form.name', required: true}/
           .row
             .large-6.columns
-              %label Skynet UUID *
+              %label UUID *
               %input{placeholder: "ad698900-2546-11e3-87fb-c560cb0ca47b", type: "text", ng_model: 'form.uuid', required: true}/
             .large-6.columns
-              %label Skynet Security Token *
+              %label Security Token *
               %input{placeholder: "4bbd2jm242dl5wmimbwz4rvlu77m0a4i", type: "text", ng_model: 'form.token', required: true}/
           .row
             .large-12.columns
-              %button.green
+              %button.green.button-like
```

Comment:
`button-like` makes the button act like a real life button - when it is clicked, it depresses slightly. But we don't want it for every button so I made it its own class.